presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,20 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent.
+* Fix ST_Buffer for tiny geometries (:issue:`13194`)
+* Fix ST_Centroid for tiny geometries (:issue:`10629`)
+* Fix failure when reading unpartitioned table from some connectors with session property ``grouped_execution_for_eligible_table_scans`` turned on.
+* Improve memory tracking for geospatial query indexing and prevent worker node from crashing due to out of memory.
+* Reduce memory usage of spatial joins by using Hilbert Packed RTree (Flatbush) instead of the JTS STR RTree. The Flatbush is a flattened tree that uses vastly fewer heap allocations, and has faster build and query times.
+
+Hive Changes
+____________
+* Add debug mode that allows reading from offline table and partition. This is controlled by session property ``offline_data_debug_mode_enabled``.
+* Fix computation for number of buckets accessed for ``max_buckets_for_grouped_execution``.
+* Fix a bug where the bucket column was not available if ``max_buckets_for_grouped_execution`` was exceeded.
+* Fix high CPU usage when writing ORC files with too many row groups.
+* Increase node-level parallelism of writing into Hive bucketed tables. To enable, set ``task.writer-count`` configuration property.
```

Comment:
cc @arhimondr Can you help fill out the release notes as suggested by @rschlussel ?

Reply:
I think the original message was 

> Writing into Hive bucketed tables now done in parallel (multiple threads per node) with respect to the task.writer-count configuration property

The idea is that previously, even when the `task.writer-count` was set to more than `1`, the partitioned inserts (for example to bucketed tables) were still running 1 thread per node. Since this release partitioned inserts become parallel and respect the `task.writer-count` same way as insert into a regular table does so.

Reply:
Maybe say something like 
```
Improve parallelism of writes to hive bucketed tables by respecting the ``task.writer-count`` configuration property
```